src/util/heap/chunk_map.rs

```diff
@@ -33,48 +28,25 @@ impl From<Chunk> for Address {
 }
 
 impl Region for Chunk {
-    const LOG_BYTES: usize = LOG_BYTES_IN_CHUNK;
+    const LOG_BYTES: usize = crate::util::heap::layout::vm_layout_constants::LOG_BYTES_IN_CHUNK;
 }
 
 impl Chunk {
     /// Chunk constant with zero address
-    const ZERO: Self = Self(Address::ZERO);
-    /// Log blocks in chunk
-    pub const LOG_BLOCKS: usize = Self::LOG_BYTES - Block::LOG_BYTES;
-    /// Blocks in chunk
-    pub const BLOCKS: usize = 1 << Self::LOG_BLOCKS;
+    pub const ZERO: Self = Self(Address::ZERO);
```

Comment:
Like `struct Block(Address)`, we may want `Chunk` to be non-zero, and use `Option<Chunk>`, too, for the same reason.  But we may postpone the refactoring to another PR because `Chunk` is not introduced in this PR.